client/web/src/enterprise/batches/list/BatchChangeStatsBar.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+
+import { mdiInformationOutline } from '@mdi/js'
+
+import { useQuery } from '@sourcegraph/http-client'
+import { Icon, LoadingSpinner, Tooltip } from '@sourcegraph/wildcard'
+
+import { GlobalChangesetsStatsResult, GlobalChangesetsStatsVariables } from '../../../graphql-operations'
+import { DEFAULT_MINS_SAVED_PER_CHANGESET } from '../../../site-admin/analytics/AnalyticsBatchChangesPage'
+import { ChangesetStatusClosed, ChangesetStatusOpen } from '../detail/changesets/ChangesetStatusCell'
+
+import { GLOBAL_CHANGESETS_STATS } from './backend'
+
+import styles from './BatchChangeStatsBar.module.scss'
+
+interface BatchChangeStatsBarProps {
+    className?: string
+}
+
+export const BatchChangeStatsBar: React.FunctionComponent<React.PropsWithChildren<BatchChangeStatsBarProps>> = () => {
+    const { data, loading } = useQuery<GlobalChangesetsStatsResult, GlobalChangesetsStatsVariables>(
+        GLOBAL_CHANGESETS_STATS,
+        {}
+    )
+
+    if (loading && !data) {
+        return (
+            <div className="text-center">
+                <LoadingSpinner className="mx-auto my-4" />
+            </div>
+        )
+    }
+    if (!data) {
+        return null
+    }
+
+    const numMinPerItemSaved: number =
+        parseInt(localStorage.getItem('MIN_PER_ITEM_SAVED_KEY') || '0', 10) || DEFAULT_MINS_SAVED_PER_CHANGESET
```

Comment:
Oops, I think you meant

```suggestion
        parseInt(localStorage.getItem(MIN_PER_ITEM_SAVED_KEY) || '0', 10) || DEFAULT_MINS_SAVED_PER_CHANGESET
```

Otherwise it won't find anything there! 🙈

Reply:
I think its supposed to be a string when using `localStorage.getItem`

Reply:
That's true, and `MIN_PER_ITEM_SAVED_KEY`'s value is the string we want to look for. 🙂 You can double check, but I'm pretty sure it won't find the value stored this way:

https://user-images.githubusercontent.com/8942601/194813737-b0f90304-d7d5-4e28-ace7-69c60225a16f.mov